packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -632,28 +632,25 @@ class HotRunner extends ResidentRunner {
         unusedElements.addAll(await unusedReport);
 
       if (unusedElements.isNotEmpty) {
+        // TODO(devoncarew): Should we only show this once during a restart session?
+        // So, show it once, and don't show it again until they've restarted?
```

Comment:
style guide says not to put in TODOs that are questions, just answer the question. :-)

In this case I think we should show it every time, because sometimes it's expected (so they won't restart) and sometimes it's not, and you don't know which it is.

Reply:
done (and, makes sense we: showing it everytime)